### PR TITLE
Feature/dev 96/broadcasting pi p while flying around

### PR DESCRIPTION
**Pull request by aminbksi**
https://momentum.nifty.pm/l/Oxm7iptF6TJj

Broadcasting PiP while flying around

### CODE REVIEW

File: packages/app/src/scenes/meeting/pages/PosBusEventsPage/PosBusEventsPage.tsx [hunk 1]
```diff
@@ -56,7 +55,7 @@ const PosBusEventsPage: FC = () => {
 
     if (liveStreamStore.isStreaming) {
       history.push(generatePath(ROUTES.collaboration.liveStream, {spaceId: space?.id}));
-    } else {
+    } else if (liveStreamStore.isLiveStreamTab) {
```
[the remark this answers is not in the document]
Comment (aminbksi) [reply packages/app/src/scenes/meeting/pages/PosBusEventsPage/PosBusEventsPage.tsx:58]: it's for when the broadcast is stopped and we are in live stream tab. so we hide the live stream tab and redirect to dashboard page

[the remark this answers is not in the document]
Comment (dmitry-yudakov) [reply packages/app/src/scenes/meeting/pages/PosBusEventsPage/PosBusEventsPage.tsx:58]: Do we know if it's ON or OFF from the received event? What if we joined the space after the broadcast started and then receive only the end event?

We don't clear isLiveStreamTab flag here and we will not clear it in other situation - I'd say detecting whether this tab is open with the store flag is not a good idea. We should somehow use the current location path.

File: packages/app/src/scenes/widgets/pages/LiveStreamPIPWidget/LiveStreamPIPWidget.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+import {Portal, SvgButton, Text} from '@momentum/ui-kit';
+import cn from 'classnames';
+
+import {ROUTES} from 'core/constants';
+import {useStore} from 'shared/hooks';
+import {VideoPanel} from 'ui-kit';
+
+import * as styled from './LiveStreamPIPWidget.styled';
+
+const Draggable: any = DraggableContent;
```
Comment (PAKovalev) [inline packages/app/src/scenes/widgets/pages/LiveStreamPIPWidget/LiveStreamPIPWidget.tsx:15]: Do we need to have this line? As I see we can use `<DraggableContent>` below.

**Review by PAKovalev — APPROVED** [review]

File: packages/app/src/scenes/meeting/pages/PosBusEventsPage/PosBusEventsPage.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (aminbksi) [reply packages/app/src/scenes/meeting/pages/PosBusEventsPage/PosBusEventsPage.tsx:58]: it is only for redirecting to dashboard only if you are in live stream tab.

**Pull request merged**